stytch/b2b/api/sessions.py

```diff
@@ -587,6 +587,7 @@ def _authenticate_jwt_local_common(
                 organization_id=org_claim["organization_id"],
                 member_id=generic_claims.reserved_claims["sub"],
                 custom_claims=custom_claims,
+                roles=roles_claim or [],
```

Comment:
All members have the `stytch_member` role (and it  can't be deleted), so I would do `roles_claim or ["stytch_member"]`

Reply:
Although I guess I'm not sure when the `roles_claim` would ever be empty, since it should always have that role